contracts/sol6/mock/MockReserveSanity.sol

```diff
@@ -0,0 +1,61 @@
+pragma solidity 0.6.6;
+
+import "../IKyberReserve.sol";
+import "../SanityRatesGasPrice.sol";
+import "../utils/Utils5.sol";
+import "../utils/zeppelin/SafeERC20.sol";
+
+
+contract MockReserveSanity is IKyberReserve, Utils5 {
+    using SafeERC20 for IERC20;
```

Comment:
could be better to just make this a mock reserve.
not only for sanity.

like mock reserve with fixed rates we use in Katalyst regression.
And the mock reserve can call sanity if address isn't 0.


but don't need to handle it now.
just name this mockReserve and add todo for trade function.

Reply:
There is already MockReserve.sol, but I forked it since I didn't know if modifying the existing one will affect the other existing tests. Anyway I can just use this instead. Just need to add more lines on sanity tests for reserve setup.